### PR TITLE
[RLlib] Add rainbow config hint to algo-documentation.

### LINKED ISSUE #7035: [RLlib] Include rainbow DQN example code
The [documentation](https://ray.readthedocs.io/en/latest/rllib-algorithms.html) mentions that rainbow DQNs can be run in RLlib, though not all settings for it are on by default. However, no where does it say what settings to change, or provide examples. This is a problem. 

@ericl If you can tell me what to do, I'll submit a PR for this this weekend after ICML

### PR DESCRIPTION
**Pull request by sven1977**
Add rainbow config hint to algo-documentation.

issue #7035 

Closes #7035

- [x] I've run `scripts/format.sh` to lint the changes in this PR.
- [x] I've included any doc changes needed for https://ray.readthedocs.io/en/latest/.
- [x] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failure rates at https://ray-travis-tracker.herokuapp.com/.

### CODE REVIEW

File: doc/source/rllib-algorithms.rst [hunk 1]
```diff
@@ -191,6 +191,17 @@ Tuned examples: `PongDeterministic-v4 <https://github.com/ray-project/ray/blob/m
 .. tip::
     Consider using `Ape-X <#distributed-prioritized-experience-replay-ape-x>`__ for faster training with similar timestep efficiency.
 
+.. hint::
+    For a cmplete `rainbow <https://arxiv.org/pdf/1710.02298.pdf>`__ setup,
```
Comment (ericl) [inline doc/source/rllib-algorithms.rst]:
```suggestion
    For a complete `rainbow <https://arxiv.org/pdf/1710.02298.pdf>`__ setup,
```

File: doc/source/rllib-algorithms.rst [hunk 2]
```diff
@@ -191,6 +191,17 @@ Tuned examples: `PongDeterministic-v4 <https://github.com/ray-project/ray/blob/m
...
+    make the following changes to the default DQN config:
+    ``"n_step": [int > 1 and << 10],
```
Comment (ericl) [inline doc/source/rllib-algorithms.rst]:
```suggestion
    ``"n_step": between 1 and 10,
```

File: doc/source/rllib-algorithms.rst [hunk 3]
```diff
@@ -191,6 +191,17 @@ Tuned examples: `PongDeterministic-v4 <https://github.com/ray-project/ray/blob/m
...
+    "batch_mode": "complete_episodes",
+    "noisy": True,
+    "num_atoms": [>1],
```
Comment (ericl) [inline doc/source/rllib-algorithms.rst]:
```suggestion
    "num_atoms": [more than 1],
```

File: doc/source/rllib-algorithms.rst [hunk 4]
```diff
@@ -191,6 +191,17 @@ Tuned examples: `PongDeterministic-v4 <https://github.com/ray-project/ray/blob/m
 .. tip::
     Consider using `Ape-X <#distributed-prioritized-experience-replay-ape-x>`__ for faster training with similar timestep efficiency.
 
+.. hint::
+    For a cmplete `rainbow <https://arxiv.org/pdf/1710.02298.pdf>`__ setup,
+    make the following changes to the default DQN config:
+    ``"n_step": [int > 1 and << 10],
+    "batch_mode": "complete_episodes",
```
Comment (ericl) [inline doc/source/rllib-algorithms.rst]: What's this for?

**Review by ericl — APPROVED** [review]

**Comment by sven1977** [comment]
All fixed.

**Review by ericl — APPROVED** [review]

**Pull request merged**